coderd/workspaceapps/request.go

```diff
@@ -353,8 +356,9 @@ func (r Request) getDatabase(ctx context.Context, db database.Store) (*databaseR
 			appSharingLevel = ps.ShareLevel
 		}
 	} else {
-		for _, app := range apps {
-			if app.Slug == r.AppSlugOrPort {
+		for _, a := range apps {
+			if a.Slug == r.AppSlugOrPort {
+				app = a
```

Comment:
Was this not fixed in a recent version of Go? Is this just to satisfy the linter?

Reply:
This is unrelated to the for-loop variable re-use Go used to have, it's for using `app` once the for loop ends.